less/index.less

```diff
@@ -49,11 +49,226 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+@ebony-color: #26303C;
+@light-grey: lightgrey;
+
+.center() {
+	display:flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.reset-text() {
+	font-size: 1.6rem;
+}
+
+.font-styles() {
+	font-family: 'Oswald', sans-serif;
+}
+
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;	
 }
 
+
+
+.container {
+	max-width: 1400px;
+	width: 100%;
+	margin: 0 auto;	
+	height: 100%;	
+	.reset-text;
+	.navbar {
+		margin-top: 3%;
+		padding-bottom: 0%;
+		background-color: white;
+		.font-styles;
+		@media ( max-width: 500px ) {
+			margin-top: 0%;
+		}
+		nav {
+			display: flex;
+			padding-bottom: 1.3%;
+			@media ( max-width: 500px) {
+				flex-direction: column;
+				padding-bottom: 0%;				
+			}
+			a {
+				text-decoration: none;
+				color: black;
+				margin: 0 auto;
+				font-size: 1.6rem;
+				@media ( max-width: 500px ) {
+					padding-bottom: 2%;
+					padding-top: 2%;
+					border-bottom: 1px solid grey;
+					width: 100%;
+					text-align: center;
+				}				
+			} // a
+			a:hover {
```

Comment:
Great job with the nesting. One thing here -- you can actually nest the :hover by using &. See example:
a {
  color: blue;
  &:hover {
    color: green;
  }
}